src/conn/api_strerror.c

```diff
@@ -69,6 +69,9 @@ __wt_wiredtiger_error(int error)
         return ("WT_CONFLICT_TABLE_LOCK: Another thread currently holds the table lock");
     case WT_CONFLICT_CHECKPOINT_LOCK:
         return ("WT_CONFLICT_CHECKPOINT_LOCK: Another thread currently holds the checkpoint lock");
+    case WT_CONFLICT_LIVE_RESTORE:
+        return (
+          "WT_CONFLICT_LIVE_RESTORE: Conflict performing operation due to running live restore");
```

Comment:
General suggestion: `due to running live restore` -> `due to an in-progress live restore`